lnrpc/lightning.proto

```diff
@@ -3711,6 +3718,22 @@ message ListPermissionsResponse {
     map<string, MacaroonPermissionList> method_permissions = 1;
 }
 
+message NodeAnnouncementUpdateRequest {
```

Comment:
> we need to decided if the request will directly substitute the current features with the ones in the request or if we need a mechanism to perform join/add/subtract of single features

Yeah this is what I was hinting at re knowing which fields are actually being updated. Seems we could try to handle the two (?) cases explicitly. So in one mode, it's a wholesale replacement of w/e feature bits we already have advertised. In the other case, a user can specify that it wants a particular bit to be added or removed. 

One possible way to do this would be to have something like an "action" defined for each entry, so something like: 
```protobuf
enum FeatureBitAction {
    ADD = 0; 
    REMOVE = 1; 
}

map<FeatureBit, FeatureBitActiion> FeatureBitUpdate = 1;
```

This way we don't force the caller to fully specify all the existing feature bits, instead they say if they want a bit to be removed or added.